pkg/assembler/graphql/schema/certifyBad.graphql

```diff
@@ -44,6 +44,18 @@ input PackageSourceOrArtifactInput {
   artifact: ArtifactInputSpec
 }
 
+"""
+PackageSourceOrArtifactInputs allows using PackageSourceOrArtifact union as
+input type to be used in bulk mutations.
+
+Exactly one list must be specified.
+"""
+input PackageSourceOrArtifactInputs {
+  packages: [PkgInputSpec!]
```

Comment:
i think matchflags should be here since it is tagged to the input that is being processed. thoughts?

Reply:
We could move it here instead of having it be a separate argument value

Reply:
Actually, no, we need these each time we can match on a package, so not only in this struct. In this case, there's no benefit between having it in each of the input structs or having it as an argument

Reply:
but would you have the case where in one we want to match a version and the other a name??

Reply:
In the case of bulk ingestion, no you could not have one match version and another match name. This also relates to this issue: https://github.com/guacsec/guac/issues/1007 where the user does not specify the `pkgMatchType` and causes a nil pointer error.